vault/request_handling.go

```diff
@@ -865,6 +865,8 @@ func (c *Core) handleRequest(ctx context.Context, req *logical.Request) (retResp
 		case logical.TokenTypeBatch:
 		case logical.TokenTypeService:
 			if err := c.expiration.RegisterAuth(ctx, &logical.TokenEntry{
+				TTL:         auth.TTL,
```

Comment:
I don't see ExpirationManager.RegisterAuth using these TokenEntry fields, what's the motivation for this part of the change?

Reply:
Are you looking on the branch? If definitely is using these, but doesn't in master.